### DISCUSSION TITLE
Test FullScreen on a different monitor

### DISCUSSION THREAD

**Discussion by Petricevic** [fyne-io/fyne#3250] [Q&A]
I have to develop a desktop program on Debian GNU / Linux bookworm / sid was a Debian 11 but I couldn't install **xorg-dev** for the library versions.
It has to be all in FullScreen. The user must never access anything except
the 6-key keypad, and see what the machine does.
The computer where I will install will have an 800x480 monitor
I am developing with a 1920x1080 laptop which will also be used for assistance.
How can I simulate this without having the monitor?
At the moment from the examples I have put together this working but very very dirty test.
```go
package main
import (
	"time"
	"fmt"
	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/driver/desktop"
	"fyne.io/fyne/v2/widget"
)

func updateTime(clock *widget.Label) {
	formatted := time.Now().Format("Time: 03:04:05")
	clock.SetText(formatted)
}

func main() {
	a := app.New()
	drv := fyne.CurrentApp().Driver()

    //Dirty, without a goes wrong but I don't use a
	fmt.Println("received", drv)
	fmt.Println("received", a)

	if drv, ok := drv.(desktop.Driver); ok {
		w := drv.CreateSplashWindow()
        // dirty, it does not appear, and it worries me a lot
		w.SetContent(widget.NewLabelWithStyle("Hello World!\n\nMake a splash!",
		fyne.TextAlignCenter, fyne.TextStyle{Bold: true}))

		clock := widget.NewLabel("")
		updateTime(clock)

		w.SetContent(clock)
		w.Resize(fyne.NewSize(800, 480))

		go func() {
			for range time.Tick(time.Second) {
				updateTime(clock)
			}
		}()
		w.ShowAndRun()
	}
}
```
Thanks in advance for any solution.

**Comment by jinzhongjia** [comment]
why don't use the function `fullscreen`?
and the 6-key keypad, means that there are only six keys on the keyboard or only six keys can be used?

**Comment by Petricevic** [comment]
I don't use  ```w.SetFullScreen(true)``` because I can't understand how it will look on a smaller monitor. Will the user be able to read the instructions? Are the icons big enough?
The keyboard has only 6 keys.

**Reply by jinzhongjia** [reply]
Fyne's components are all vector scaling, so your concerns about different renderings should not arise

**Reply by jinzhongjia** [reply]
But I suggest you get it on an actual device and test it

**Comment by Jacalz** [comment]
The numerical entry tutorial might be interesting for the keypad: https://developer.fyne.io/extend/numerical-entry

**Reply by Jacalz** [reply]
Also, FYI, you can change the driver code above to this instead:
```
a := app.New()
drv := a.Driver()
```

**Reply by Petricevic** [reply]
Thank you! Works.

**Reply by Jacalz** [reply]
To simulate the screen size, you can set the window size `w.Resize()` to the same ratio as the other screen (not necessarily same size as the size isn't pixels). The scaling of Fyne should make it look basically the same.

If you want a better simulation, your best bet might be to force the screen resolution in the OS settings, if possible, to the same values.

**Comment by andydotxyz** [comment]
Resizing the window doesn't seem like a dirty test. 
The alternative is to reconfigure your monitor to display the desired resolution for your app ok fullscreen.
